README.md

```diff
@@ -4,30 +4,24 @@ This is the repository for the backend file format check for the [Transfer Digit
 
 The file format checks run in AWS Lambda. They run the [DROID] file format tool to extract [PRONOM] IDs.
 
-The Lambda function connects to an AWS EFS file store which contains both DROID and the files to be scanned.
+The Lambda function downloads the file passed as input to the lambda local file system. 
 
-DROID and its file signatures are deployed to EFS by an AWS ECS task.
+The function then calls the Droid API against this file to extract the [PRONOM] ids 
 
 [Transfer Digital Records]: https://github.com/nationalarchives/tdr-dev-documentation/
 [DROID]: https://www.nationalarchives.gov.uk/information-management/manage-information/preserving-digital-records/droid/
 [PRONOM]: http://www.nationalarchives.gov.uk/PRONOM/Default.aspx
-[Run ECS Task]: https://github.com/nationalarchives/tdr-file-format/actions/workflows/run.yml
-[build GitHub actions job]: https://github.com/nationalarchives/tdr-file-format/actions/workflows/build.yml
 [deploy GitHub actions job]: https://github.com/nationalarchives/tdr-file-format/actions/workflows/deploy.yml
 
 ## Deployment
 
 ### DROID and file format signatures
 
-The [build GitHub actions job] will run when a pull request is merged to master. 
-The job will check to see if there are any changes to the Dockerfile. 
-If there are, it will build a new version of the docker image and push it to ECS.
-
-To run the new image, run the [Run ECS Task] workflow with the latest docker image version and the environment you are deploying to.
+Droid is contained within the `"uk.gov.nationalarchives" % "droid-api"` dependency. 
+The binary and container signature files are downloaded when the lambda is first started. 
+The versions of the signature files are stored in the `DROID_SIGNATURE_VERSION` and `CONTAINER_SIGNATURE_VERSION` environment variables. 
+To update to a new version of these signature files, we need to update the values in the environment variables.
```

Comment:
Is this a manual process to find out if there is a new version? Probably should note where to find the latest versions.

Reply:
It is a manual process at the moment. I'm hoping there will be some kind of message we can subscribe to to automatically update them but there's nothing yet. I've put this in the README along with where to find the signature files.